wp-graphql.php

```diff
@@ -684,6 +690,45 @@ public static function get_app_context() {
 
 			return $app_context;
 		}
+
+		/**
+		 * Adds default GraphQL post-types settings
+		 *
+		 * @param WP_Post_Type|array $args       Post type arguments.
+		 * @param string             $post_type  Post type
+		 * 
+		 * @return array
+		 */
+		public static function add_post_type_settings( $args, $post_type ) {
+			global $wp_post_types;
+
+			$post_type_settings = [
+				'show_in_graphql'                 => true,
+				'graphql_single_name'             => str_replace( '_', '', ucwords( strtolower( $post_type ), '_' ) ),
+				'graphql_plural_name'             => str_replace( '_', '', ucwords( strtolower( $post_type ), '_' ) ) . 's',
+				'register_schema_type'            => true,
+				'register_root_query'             => true,
+				'register_root_connection'        => true,
+				'connect_to_comments'             => true,
+				'connect_to_taxonomies'           => true,
+				'supports_content_node'           => true,
+				'supports_post_object_union'      => true,
+				'supports_menu_item_object_union' => true,
```

Comment:
WordPress also has rules for this. 

`show_in_nav_menus`: https://developer.wordpress.org/reference/functions/register_post_type/#show_in_nav_menus

If a CPT is set to show in Nav Menus, WPGraphQL should support Nav Menu Items returning that type.

Reply:
I know, but this is just determine if it's displayed that way in the schema tree.